core/modules/comment/comment.install

```diff
@@ -352,6 +351,20 @@ function comment_update_1002() {
 }
 
 /**
+ * Node type comment settings should be closed instead of hidden by default.
+ */
+function comment_update_1003() {
+  foreach (node_type_get_names() as $type => $name) {
```

Comment:
For the best compatibility and lowest likelihood of getting incorrectly cached data, could we switch this to `foreach (config_get_names_with_prefix('node.type.') as $config_name)`?